ThirdPartyAdapters/vungle/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 ## Vungle Android Mediation Adapter Changelog
 
-#### Version 6.7.0.0
-- Verified compatibility with Vungle SDK 6.7.0.
+#### Version 6.7.1.0
```

Comment:
please don't update this live version information and insert new version information ahead.